### ISSUE TITLE
"Not enough Credits" when selling something for negative price

### ISSUE THREAD

**Issue by D3nnis3n** [StrangeLoopGames/EcoIssues#16796]
```
I was really surprised when I saw that you can put a negative value to buy/sell items in the store. I thought it was really cool, and immediately thought about tailings. The idea was, a person can pay others to take tailings off their hands by putting it in their sell list with a negative price, and other players can offer to handle said tailings by putting them in their buy list, again at a negative price, wherein they would be paying them to handle the disposal. Unfortunately, it does not work the way I expected it to. My friend tried to buy tailings from my store, with the price set as negative 25, having none of my credits at the time. Instead of allocating credits to him for the sale, the "no credits"/"not enough credits" message came up.

Do you guys think this is something you would likely include, or at least fix into something usable?

I will do some more testing with it, but for the meantime it looks like it isn't going to work like I had hoped it would... :cry:
```

TL;DR:
When you sell Tailings in your Shop for a negative value and someone wants to buy them but does not have any of the currency you get "Not enough Credits", despite the negative amount means you would get credits. The check should be adapted for that.

**Comment by MishkaFur** [comment]
Cannot do negative values in 9.0 1611, someone tried (i think Xao?) to do a order to "buy" tailings off people and store them safely.

Just does not accept the - input from keyboard and cannot use the + and - buttons to end up minus either.

**Comment by D3nnis3n** [comment]
That's even more bug.
